http-clients/src/main/java/com/palantir/remoting/http/errors/SerializableErrorErrorDecoder.java

```diff
@@ -101,18 +106,16 @@ public Exception decode(String methodKey, Response response) {
         }
     }
 
-    private static Exception constructException(Class<? extends Exception> exceptionClass, String message) {
-        try {
-            return exceptionClass.getConstructor(String.class).newInstance(message);
-        } catch (Exception e1) {
-            return new RuntimeException(String.format(
-                    "Failed to construction exception of type %s, constructing RuntimeException instead: %s%n%s",
-                    exceptionClass.toString(), e1.toString(), message));
+    private static Exception constructException(
```

Comment:
nope. we can release 0.7.0 today. it's a major breaking change, both in
terms of code and config.

On Thu, Feb 18, 2016 at 2:54 PM Jason Zhao <notifications@github.com> wrote:

> In
> http-clients/src/main/java/com/palantir/remoting/http/errors/SerializableErrorErrorDecoder.java
> <https://github.com/palantir/http-remoting/pull/45#discussion_r53396621>:
>
> > @@ -101,18 +106,16 @@ public Exception decode(String methodKey, Response response) {
> >          }
> >      }
> >
> > -    private static Exception constructException(Class<? extends Exception> exceptionClass, String message) {
> > -        try {
> > -            return exceptionClass.getConstructor(String.class).newInstance(message);
> > -        } catch (Exception e1) {
> > -            return new RuntimeException(String.format(
> > -                    "Failed to construction exception of type %s, constructing RuntimeException instead: %s%n%s",
> > -                    exceptionClass.toString(), e1.toString(), message));
> > +    private static Exception constructException(
>
> oh doh, thanks adam! neither of these changes are released yet right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/palantir/http-remoting/pull/45/files#r53396621>.
>